docker/jstltck.sh

```diff
@@ -97,6 +97,11 @@ cd $TCK_HOME/$GF_TOPLEVEL_DIR/glassfish/bin
 ./asadmin create-jvm-options -Djavax.xml.accessExternalStylesheet=all
 ./asadmin create-jvm-options -Djavax.xml.accessExternalSchema=all
 ./asadmin create-jvm-options -Djavax.xml.accessExternalDTD=file,http
+#https://github.com/eclipse-ee4j/jakartaee-tck/issues/631
+if [[ "$JDK" == "JDK11" || "$JDK" == "jdk11" ]]; then
```

Comment:
I think this check if correct.  

I just wanted to mention that I still want to make it possible for later than JDK11 be supported via https://github.com/eclipse-ee4j/jakartaee-tck/issues/156 but I am still working on that.  To point that if/when https://github.com/eclipse-ee4j/jakartaee-tck/issues/156 is solved, we will need to change `==` to `>=` which likely means we will need to change the value to a number at that point as well (perhaps the bash script could just ignore the first 3 letters as a workaround).